lib/tasks/chat_message.rake

```diff
@@ -49,3 +49,86 @@ def rebake_chat_message(chat_message, opts = {})
 rescue => e
   puts "", "Failed to rebake chat message (chat_message_id: #{chat_message.id})", e, e.backtrace.join("\n")
 end
+
+task 'chat:make_channel_to_test_archiving', [:user_for_membership] => :environment do |t, args|
```

Comment:
Not sure if this part really needs review, it's just to make test channels in local development.